css/index.css

```diff
@@ -1,3 +1,300 @@
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
 
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+body{
+	
+	margin: 3% 3% 3% 3%;
+	
+}
+
+section{
+	
+	
+
+
+}
+
+.block{
+
+	margin: 15px;
+	
+}
+.block p{
+	
+}
+
+h4{
+	font-size: 18px;
+	font-weight: bold;
+}
+
+header{
+	display: flex;
+	height: 5%;
+	justify-content: space-between;
+	
+}
+		
+	header nav{
+		display: flex;
+		
+		
+	}
+		header nav a{
+			margin:20px 25px 25px 25px;
+			color:black;
+			text-decoration-line: none;
+			justify-content: space-evenly;
+		}
+	header a{
+		display: flex;				
+	
+		align-items: center;
+		width: 20%;
+					
+		}
+	
+
+
+#bar1{
+	display: flex;
+	justify-content: space-around;
+	height: 35%;
+}		
+
+	#bar1 #text{
+		display: flex;
+		flex-direction: column;
+		align-items: center;
+	}
+		#bar1 h1{
+			padding: 100px 0px 0px 0px;
+			text-align: center;
+			font-size: 75px;
+			width: 75%;
+		}
+		#bar1 button{
+		
+			font-size: 18px;
+			
+	
+			
+		
+		}
+	#bar1 img{
+		margin-top: 30px;
+       
+        height: 80%;
+		
+		
+	}
+
+
+#middle-img, #services-header-img{
+	width:100%;
+}
+#bar2{
+	display: flex;
+    border-top: 1px solid black;
+   
+}
+#bar4{
+	display: flex;
+}
+footer{
+	border-top: 1px solid black;
+}
+
+footer p{
+	font-size: 15px;
+	margin-top: 15px;
+}
+#copyright{
+	
+		font-weight: bold;
+		text-align: center;
+		font-size: 15px;
+		padding: 0px ;
+}
+
+@media( max-width: 500px){
+
+body{
+    margin:0px 0px 0px 0px;
+    padding: 0px;
+}
+    header{
+        display: flex;
+        flex-direction: column-reverse;
+        align-items: center;
+        margin-bottom: 0%;
+        width: 100%;
+        height: 40%;
+    }
+
+    header nav{
+        margin-left: 0px;
+        display: flex;
+        flex-direction: column; 
+        width: 100%;
+        margin-right: 50px;
+    }
+
+        header nav a{
+            width: 100%;
+            border-top: 1px solid gray;
+            
+        }
+
+    #bar1 img{
+        display: none;
+    }
+
+#text h1{
+    width: 100%;
+    font-size: 50px;
+}
+#bar2{
+    display: flex;
+    flex-direction: column;
+    margin-bottom: 10px;
+}
+#middle-img{
+    display: flex;
+   
+}
+
+#bar4{
+    display: flex;
+    flex-direction: column;
+}
+.block{
+
+}
+
+footer{
+margin-top: 30%;
+}
+
+#text{
+    height: 30%;
```

Comment:
Removing the `height` here will allow the text of the button to sit inside the button container.